src/page-modules/departures/details/index.tsx

```diff
@@ -0,0 +1,68 @@
+import LineChip from '@atb/components/line-chip';
+import { Typo } from '@atb/components/typography';
+import style from './details.module.css';
+import { ButtonLink } from '@atb/components/button';
+import { ColorIcon, MonoIcon } from '@atb/components/icon';
+import { useRealtimeText } from './use-realtime-text';
+import { addMetadataToEstimatedCalls } from './utils';
+import { ServiceJourneyData } from '../server/journey-planner/validators';
+import { EstimatedCallRows } from './estimated-call-rows';
+import { PageText, useTranslation } from '@atb/translations';
+
+export type DeparturesDetailsProps = {
+  fromQuayId?: string;
+  serviceJourney: ServiceJourneyData;
+};
+
+export function DeparturesDetails({
+  fromQuayId,
+  serviceJourney,
+}: DeparturesDetailsProps) {
+  const { t } = useTranslation();
+  const focusedCall =
+    serviceJourney.estimatedCalls.find((call) => call.quay.id === fromQuayId) ||
+    serviceJourney.estimatedCalls[0];
+  const title = `${serviceJourney.line.publicCode} ${focusedCall.destinationDisplay.frontText}`;
+  const realtimeText = useRealtimeText(serviceJourney.estimatedCalls);
+  const estimatedCallsWithMetadata = addMetadataToEstimatedCalls(
+    serviceJourney.estimatedCalls,
+    fromQuayId,
+    undefined,
+  );
+
+  return (
+    <section className={style.container}>
+      <div className={style.headerContainer}>
+        <ButtonLink
+          mode="transparent"
+          href={`/departures/${focusedCall.quay.stopPlace.id}`}
+          title={t(PageText.Departures.details.back)}
+          icon={{ left: <MonoIcon icon="navigation/ArrowLeft" /> }}
+        />
+        <div className={style.header}>
+          <LineChip
+            transportMode={serviceJourney.transportMode}
+            transportSubmode={serviceJourney.transportSubmode}
+            publicCode={serviceJourney.line.publicCode}
+          />
+          <Typo.h2 textType="heading--big">{title}</Typo.h2>
```

Comment:
Should there be any accessibility label for the header of the page, saying something about the chosen stop place and potentially the real time information? Not sure how it should be done when theres no natural focusable element.

Reply:
I was thinking about this and I think we should. I'm not sure how to solve this either, and I see that we are missing this from the other pages as well 🤔 Should we make an issue to find a solution for this and update where needed?

Reply:
Sure, that works.

Reply:
https://github.com/AtB-AS/kundevendt/issues/15431